File: 0001 Code Chef/0013 Thala For a Reason/implementation/0004 java/ThalaForReason.java
import java.util.Scanner;


public class ThalaForReason{
    public static void main(String[] args){
        Scanner console = new Scanner(System.in);
        String stringInput = console.nextLine();
        int number = Integer.parseInt(stringInput);
        if(number == 7){
            System.out.println("THALA");
        }else{
            System.out.println("SADGE");
        }
    }
}

File: 0001 Code Chef/0013 Thala For a Reason/implementation/0009 dart/thala_for_reason.dart
import 'dart:io';

void main(){
  String? stringInput = stdin.readLineSync();
  if(stringInput != null){
    int? number = int.tryParse(stringInput);
    if(number != null){
      if(number == 7){
        print("THALA");
      }else{
        print("SADGE");
      }
    }
  }
}